classes/author.rb

```diff
@@ -0,0 +1,13 @@
+require 'item'
+
+class Author
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
```

Comment:
According to the requirements, you need to add a property **items** that is an array of **Item** objects